composer.json

```diff
@@ -56,7 +56,7 @@
 	"require" : {
 		"clearFw/clearFw" : "~1.0",
 		"easyrdf/easyrdf" : "0.9.*",
-		"doctrine/dbal" : "^2.5",
+		"doctrine/dbal" : "~2.5.0",
```

Comment:
Unfortunately we deploy with min stability dev, so we get the unreleased 2.6. So this fix is suitable.